lib/presentation/home_page/components/horizontal_weather_list.dart

```diff
@@ -34,19 +39,20 @@ class HorizontalWeatherList extends StatelessWidget {
             children: [
               //TODO zmienic 7200 na czas lokalny/offset lokalny
               // currentTime< itemTime
```

Comment:
Delete to do after done :) by the way, try //TODO: , your IDE should highlights that as a warnings and it will be easier to remember about that.